lib/kitchen/verifier/busser.rb

```diff
@@ -173,9 +173,16 @@ def gem_install_args
         gem, version = config[:version].split("@")
         gem, version = "busser", gem if gem =~ /^\d+\.\d+\.\d+/
 
+        root = config[:root_path]
+        gem_bin = remote_path_join(root, "bin")
+
+        # We don't want the gems to be installed in the home directory,
+        # this will force the bindir and the gem install location both
+        # to be under /tmp/verifier
         args = gem
         args += " --version #{version}" if version
-        args += " --no-rdoc --no-ri --no-format-executable"
+        args += " --no-rdoc --no-ri --no-format-executable -n #{gem_bin}"
```

Comment:
I'll verify this, but I think we're going to clobber over the bin stub when we run `busser setup` which runs a [nuke-the-world-of-rubyness purge](https://github.com/test-kitchen/busser/blob/master/lib/busser/command/setup.rb#L105-L128) of its environment to make sure that we're not gem-activating anything by accident or coincidence. Worst case is we might have to adjust the `-n #{gem_bin}` target.